File: src/net/eduard/api/command/SetSkinCommand.java
package net.eduard.api.command;

import net.eduard.api.lib.modules.Mine;
import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.manager.PlayerSkin;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class SetSkinCommand extends CommandManager {
    public SetSkinCommand() {
        super("setskin");

    }

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (args.length ==0){
            sendUsage(sender);
        }else{
            if (Mine.onlyPlayer(sender)) {
                Player p = (Player) sender;
                String playerName = args[0];
                sender.sendMessage("§aSua skin foi alterada para " + playerName);
                PlayerSkin.change(p, playerName);
            }
        }
        return true;
    }
}

File: src/main/java/net/eduard/api/command/SetSkinCommand.kt
package net.eduard.api.command

import net.eduard.api.core.PlayerSkin
import net.eduard.api.lib.manager.CommandManager
import org.bukkit.entity.Player

class SetSkinCommand : CommandManager("setskin","setplayerskin") {
    init{
        description= "Defina uma Skin para si mesmo"
        usage= "/<command> <playerName>"
    }
    override fun playerCommand(player: Player, args: Array<String>) {
        if (args.isEmpty()) {
            sendUsage(player)
            return
        }
        val playerName = args[0]
        player.sendMessage("§aSua skin foi alterada para $playerName")
        PlayerSkin.change(player, playerName)
    }

}